### PR TITLE
Standing on giants - ported exercise from ecmascript track

### PR DESCRIPTION
**Pull request by VestigeJ**
Used the two-bucket exercise from ecmascript to port the problem over for typescript.

### CODE REVIEW

**Comment by masters3d** [comment]
"-> An implementation for 'two-bucket' was found, but config.json does not reference this exercise.
"

File: exercises/two-bucket/two-bucket.example.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        measurements = [j, k]
+        moveCount++
+        prBool = !prBool
+      }
+      return moveCount
+    }
+    moves() {
+      let j = 0,
+        k = 0 // j will be running val of bucket one, k = running val of bucket two
+      this.starter === 'one' ? j = this.x : k = this.y
```
Comment (masters3d) [inline exercises/two-bucket/two-bucket.example.ts]: starter should be an enum. 
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html

File: config.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
```
Comment (masters3d) [inline config.json]: This one should not change. You need to add a config file to your exercise.

Comment (masters3d) [reply config.json]: I am sorry. I was wrong.  You do need to change it.

File: exercises/two-bucket/two-bucket.example.ts (hunk 1 above)
Comment (masters3d) [reply exercises/two-bucket/two-bucket.example.ts]: I update the track to use the latest TS which has support to string enums.

**Comment by VestigeJ** [comment]
Sorry for the delay team - we just had our second baby 👶

**Comment by masters3d** [comment]
Congrats! No rush. I am just marking these so I know what they status is.

**Comment by masters3d** [comment]
Reopened this as a new PR https://github.com/exercism/typescript/pull/123

**Pull request closed without merging**